src/library/Cardano/CoinSelection/Fee.hs

```diff
@@ -181,28 +166,40 @@ adjustForFee
     => FeeOptions i o
     -> CoinMap i
     -> CoinSelection i o
-    -> ExceptT ErrAdjustForFee m (CoinSelection i o)
+    -> ExceptT FeeAdjustmentError m (CoinSelection i o)
 adjustForFee unsafeOpt utxo coinSel = do
     let opt = invariant
             "adjustForFee: fee must be non-null" unsafeOpt (not . nullFee)
     senderPaysFee opt utxo coinSel
   where
     nullFee opt = estimateFee (feeEstimator opt) coinSel == Fee C.zero
 
--- | The sender pays fee in this scenario, so fees are removed from the change
+--------------------------------------------------------------------------------
+-- Internal Functions
+--------------------------------------------------------------------------------
+
+-- Calculates the current fee associated with a given 'CoinSelection'.
+--
+-- If the result would be less than zero, returns 'Nothing'.
```

Comment:
You mean: 
`If the result is less than zero, then 'Nothing' is returned.`
?

Reply:
Thanks for the feedback on this comment.

Actually, either phrasing would work, since the "would" here expresses a hypothetical (the _returned_ result can never be less than zero).

Though I'll change it to your suggested wording, as I agree it's shorter and clearer! :+1: 

Fixed in 8e42eca4e1612b50f41a553f7bb6fa8fa97dc7a8.